search-app/src/cmr/search/results_handlers/variables_json_results_handler.clj

```diff
@@ -0,0 +1,30 @@
+(ns cmr.search.results-handlers.variables-json-results-handler
+  "Handles extracting elasticsearch variable results and converting them into a JSON search response."
+  (:require
+   [cheshire.core :as json]
+   [cmr.common-app.services.search :as qs]
+   [cmr.common-app.services.search.elastic-results-to-query-results :as elastic-results]
+   [cmr.common-app.services.search.elastic-search-index :as elastic-search-index]))
+
+(defmethod elastic-search-index/concept-type+result-format->fields [:variable :json]
+  [concept-type query]
+  ["concept-id" "provider-id" "native-id" "variable-name" "measurement"])
+
+(defmethod elastic-results/elastic-result->query-result-item [:variable :json]
+  [context query elastic-result]
+  (let [{{[variable-name] :variable-name
```

Comment:
This weird syntax is because the result coming back from elasticsearch is weird.